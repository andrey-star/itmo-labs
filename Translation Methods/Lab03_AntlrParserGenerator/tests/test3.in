def f(a, b, c):
	d = 5
	e = a + c
	return a + b / -(d + e)

a = f(1, 2, 3)
b = f(1, -2, 3)
c__8908 = f(1, 2, -(3))

def g():
	a = f(1, 2, 3)
	b = int(input())
	return a + b
