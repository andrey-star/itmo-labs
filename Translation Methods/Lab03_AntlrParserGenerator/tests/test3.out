int f(int a, int b, int c);
int g();

int f(int a, int b, int c) {
	int d = 5;
	int e = a + c;
	return a + b / -(d + e);
}

int g() {
	int a = f(1, 2, 3);
	int b;
	scanf("%d", &b);
	return a + b;
}

int main() {
	int a = f(1, 2, 3);
	int b = f(1, -2, 3);
	int c__8908 = f(1, 2, -(3));
	return 0;
}
